app/views/repos/new.html.slim

```diff
@@ -18,13 +18,13 @@
 
   .tab-content
     .tab-pane.active#own
-      = render 'shared/add_repos_list', repos: @own_repos
+      = render 'shared/add_repos_list', repos: @own_repos, all_repos: Repo.all
```

Comment:
I think we can skip passing all repos. We can calculated them in the model method itself if needed.